mmcls/models/classifiers/image.py

```diff
@@ -77,7 +77,7 @@ def forward_train(self, img, gt_label, **kwargs):
 
         return losses
 
-    def simple_test(self, img, img_metas):
+    def simple_test(self, img):
```

Comment:
Why remove `img_metas`?  better not change the original interface.

Reply:
If i do not remove this option, pytorch2onnx.py will report an error when running.

Reply:
I try to modifying pytorch2onnx.py to avoiding change the original code.